Document/0x05b-Basic-Security_Testing.md

```diff
@@ -281,7 +281,120 @@ Frida also provides bindings for various languages, including Python, C, NodeJS,
 
 ##### r2frida
 
--- ToDo: <https://github.com/OWASP/owasp-mstg/issues/1232>
+[r2frida](https://github.com/nowsecure/r2frida "r2frida on Github") is a project that allows radare2 to connect to Frida, effectively merging the powerful reverse engineering capabilities of radare2 with the dynamic instrumentation toolkit of Frida. R2frida allows you to:
+
+- Attach radare2 to any local process or remote frida-server via USB or TCP.
+- Read/Write memory from the target process.
+- Load Frida information such as maps, symbols, imports, classes and methods into radare2.
+- Call r2 commands from frida as it exposes the r2pipe interface into the Frida Javascript API.
+
+###### Installing r2frida
+
+Before installing r2frida, make sure you have installed radare2 on your system. In GNU/Debian you also need to install the following dependencies:
+
+```shell
+$ sudo apt install -y make gcc libzip-dev nodejs npm curl pkg-config git
+```
+
+Once the dependencies are installed, you may proceed to install r2frida. The recommended installation method is via r2pm. To install with `r2pm`, simply run the following command:
+
+```shell
+$ r2pm -ci r2frida
+```
+
+However, you may also compile r2frida yourself in the traditional way. You can find how to do this in the official documentation of [r2frida](https://github.com/nowsecure/r2frida/blob/master/README.md "r2frida readme").
+
+###### Using r2frida
+
+With frida-server running, you should now be able to attach to it using the pid, spawn path, host and port and USBID/pid. For example, to attach to PID 1234:
+
+```shell
+$ r2 frida://1234
+```
```

Comment:
Maybe you can add:

For more examples see the [usage section in the r2frida repo](https://github.com/nowsecure/r2frida/blob/master/README.md#usage "Usage of r2frida")